views/layouts/main.handlebars

```diff
@@ -77,6 +82,13 @@
               </div>
               {{/if}}
             </div>
+            <p>
```

Comment:
I thought we already deleted those lines? (show languages in text)

Reply:
@wildan3105 Should I remove the `<p> ..... </p>` and send PR again ? ?